stake/src/epoch.rs

```diff
@@ -1,8 +1,9 @@
+use serde::{Deserialize, Serialize};
 use tendermint::block;
 
 /// Epoch represents a given epoch for Penumbra and is used
 /// for calculation of staking exchange rates.
-#[derive(Debug, Eq, PartialEq, Copy, Clone)]
+#[derive(Debug, Eq, PartialEq, Copy, Clone, Serialize, Deserialize)]
```

Comment:
We should be careful to use the `try_from` / `into` Serde annotations to make sure that we always serialize *through* the protobuf types, and don't end up accidentally defining a serialization format from our internal implementation details.